From 02b3f69dc0a26473decbf1cb30d12f59cd2855e2 Mon Sep 17 00:00:00 2001
From: Emmanuel Charpentier <emm.charpentier@free.fr>
Date: Fri, 13 Oct 2017 08:23:23 +0200
Subject: [PATCH 8/8] rcmd_exec

On Cygwin some of the scripts in $R_HOME/bin can fail to be recognized
as executable, because they do no contain a shebang line and, depending
on the ACL settings in the Cygwin mount, may not have an executable flag
either.  This results in the scripts not being run properly.  It's fine
to just check that they exist.  See https://trac.sagemath.org/ticket/20655

---
 src/scripts/Rcmd.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scripts/Rcmd.in b/src/scripts/Rcmd.in
index 76d78d5..4b92483 100644
--- a/src/scripts/Rcmd.in
+++ b/src/scripts/Rcmd.in
@@ -50,7 +50,7 @@ case "${1}" in
     exit 1
     ;;
   *)
-    if test -x "${R_HOME}/bin/${1}"; then
+    if test -f "${R_HOME}/bin/${1}"; then
       cmd="${R_HOME}/bin/${1}"
     else
       cmd="${1}"
-- 
2.14.2

